modules/test-runner/src/transfer/asyncTransfer.test.ts

```diff
@@ -27,30 +34,141 @@ describe("Async Transfers", () => {
   }, 90_000);
 
   test("happy case: client A transfers eth to client B through node", async () => {
-    const transfer: AssetOptions = { amount: ETH_AMOUNT_SM, assetId: AddressZero };
```

Comment:
can we add this back just to make sure the amount and assetId is this the same throughout the test

Reply:
just the `transfer` object? we can if you think it makes it easier to read, i personally prefer it the other way

keeping the `fundChannel` though because it checks to make sure money is deposited by the client correctly

Reply:
I find it easier to read but mostly it makes sure the `amount` and `assetId` is not hardcoded in multiple places.

Yeah we should use the `fundChannel` in other tests too.